spec/furniture/marketplace/checkouts_controller_request_spec.rb

```diff
@@ -10,9 +10,10 @@
     let(:checkout) { create(:marketplace_checkout, cart: cart, shopper: cart.shopper)}
     context "when a stripe_session_id is in the params" do
       it "marks the cart as checked out" do
-        get polymorphic_path([space, room, marketplace, checkout], { stripe_session_id: "12345" })
+        get polymorphic_path(checkout.location, { stripe_session_id: "12345" })
         expect(cart.reload).to be_checked_out
         expect(checkout.reload).to be_paid
+        expect(response).to redirect_to(checkout.becomes(Marketplace::Order).location)
```

Comment:
The changes in this PR seem like a large enough refactor that I assumed there'd be more unit tests added/modified to provide test coverage for the new Order model, corresponding Controllers, and orders/show, orders/_order. But maybe it's just the over tester in me.

Reply:
Can you clarify where you're seeing risk here? At this point, we don't have significant deal-flow; so flexibility in mutating the design seems more valuable than enshrining the design via tests.

Reply:
that's a good point. 👍

Reply:
I think there are *definitely* risks here, I just don't know which ones are worth writing tests for yet! If you have ideas about which particular cases you'd like tested (either cross-request or individual request) I would be happy to implement them!